README.md

```diff
@@ -1,8 +1,31 @@
 # Redux-cycles
+
+<div align="center">
+  <img src="logo.png" alt="Cycle.js + Redux = <3" />
```

Comment:
I've thought about that... but the Cycle logo isn't symmetric, its right side looks cut out, so I thought the logo would look better if the plus sign kind of filled that cut; having it in the middle would probably look odd-ish.

What do you mean by pulling both of the logos? I already made a picture by combining these. If you mean we should have two image tags and plus, equals, and a heart as text, I'd say formatting that will be terrible on its own. The people wouldn't also be able to easily save the logo to use it elsewhere (like in a tweet)